config.toml

```diff
@@ -138,10 +138,10 @@ time_format_default = "January 02, 2006 at 3:04 PM PST"
 description = "Production-Grade Container Orchestration"
 showedit = true
 
-latest = "v1.25"
+latest = "v1.26"
```

Comment:
Something is wrong with this PR. Why are we including the config.toml file in this commit?

Reply:
Whoops, someone I got an errant `config.toml` commit rebased in with my branch. Removed and squashed.